### PR TITLE
Suport sorting by variable length, not just full length of line.

### PR DESCRIPTION
**Pull request by labithiotis**
**From**
```javascript
const os = require('os');
const Koa = require('koa');
const _ = require('lodash');
const cookie = require('cookie');
const stream = require('stream');
const route = require('koa-route');
const dnscache = require('dnscache');
const sleep = require('sleep-promise');
```

**To**
```javascript
const _ = require('lodash');
const os = require('os');
const Koa = require('koa');
const route = require('koa-route');
const sleep = require('sleep-promise');
const cookie = require('cookie');
const stream = require('stream');
const dnscache = require('dnscache');
```

### CODE REVIEW

File: lib/sort-lines.js [hunk 1]
```diff
@@ -52,6 +52,18 @@ function lineLengthReverseCompare(a, b) {
   return a.length > b.length ? -1 : 1;
 }
 
+function getVariableCharacters(line) {
+  return (line.match(/(.*)=/) || []).pop();
+}
+
+function variableLengthCompare(a, b) {
+  return getVariableCharacters(a).length > getVariableCharacters(b).length ? 1 : -1;
```
Comment (Tyriar) [inline lib/sort-lines.js]: [].pop().length will throw

File: lib/sort-lines.js [hunk 2]
```diff
@@ -52,6 +52,18 @@ function lineLengthReverseCompare(a, b) {
   return a.length > b.length ? -1 : 1;
 }
 
+function getVariableCharacters(line) {
+  return (line.match(/(.*)=/) || []).pop();
```
Comment (Tyriar) [inline lib/sort-lines.js]: Won't this be weird if the variables are a mix of const/var?

```js
const a = 1;
const b = 11;
var c = 2;
var d = 22;
```

Comment (labithiotis) [reply lib/sort-lines.js]: Why would anyone mix `const` and `var`? Maybe `let` and `const`, but arguably separating them makes the code more readable. For example, I think this is awful:
```
let x = 1;
const b = 2;
const def = 4;
let y = 423523;
```

vs 
```
let x = 1;
let y = 423523;
const b = 2;
const def = 4;
```

File: lib/sort-lines.js (hunk 1 above)
Comment (labithiotis) [reply lib/sort-lines.js]: Yer I guess I could fallback to empty string in `getVariableCharacters`

**Comment by labithiotis** [comment]
Regarding the usefulness, I find it far more readable to read the variable before the `=` instead of the string in require('').

**Comment by labithiotis** [comment]
@Tyriar Is this PR all good?

**Pull request merged**